
syntax = "proto3";

package twitteranomalydetection;


service TwitterAnomalyDetection {

  rpc DetectAnomalies (Request) returns (Dictionary) {}
}

message Request {
  string raw_json = 1;
  string only_last = 2;
  bool resampling = 3; 

}

message Dictionary {
  map<string, float> pairs = 1;
}
